frontend-react/src/pages/data-dashboard/DataDashboard.tsx

```diff
@@ -0,0 +1,56 @@
+import { GridContainer } from "@trussworks/react-uswds";
```

Comment:
I *think* this is the only file that should be within `/pages/data-dashboard/` and the rest of these components should live within `/components`

Reply:
@jpandersen87 are you ok with that?

Reply:
👍